mediathread/urls.py

```diff
@@ -172,8 +172,8 @@
          name='affil_activate'),
 
     # New course-aware routes
-    path('course/<int:course_pk>/react/asset/<int:pk>/',
-         ReactAssetDetailView.as_view(),
+    path('course/<int:course_pk>/asset/<int:pk>/',
```

Comment:
Looking through this a bit...is there an *easy* way to preserve old functionality, e.g. like the "oldhome" setup. Largely as reference for QA efforts.